pkg/validation/validate/schema.go

```diff
@@ -99,6 +99,9 @@ func NewSchemaValidator(schema *spec.Schema, rootSchema interface{}, root string
 // SetPath sets the path for this schema validator
 func (s *SchemaValidator) SetPath(path string) {
 	s.Path = path
+	for _, v := range s.validators {
+		v.SetPath(path)
+	}
```

Comment:
would like to see a test case where this goes wrong before the PR and is fixed afterwards.

Reply:
@sttts there is a pretty comprehensive write up of exactly what is happening, along with an example from before and after the fix in https://github.com/kubernetes/kube-openapi/issues/262. Are there additional cases that would be more helpful for review?

Reply:
Indeed comprehensive :)